Functions-request-config.js

```diff
@@ -40,6 +41,11 @@ const requestConfig = {
   args: ["1", "bitcoin", "btc-bitcoin", "btc", "1000000", "450"],
   // expected type of the returned value
   expectedReturnType: ReturnType.uint256,
+  // Redundant URLs which point to encrypted off-chain secrets
+  secretsURLs: [],
+  // Per-node offchain secrets objects used by the `functions-build-offchain-secrets` command
+  // The first entry will be used by the simulator if `secrets` is undefined
```

Comment:
I think we need a better explanation. Something like "instead of a single set of secrets sent to all nodes, you can specify per-node secrets. To do so, leave 'secrets' undefined and define this map. The keys are .."

Reply:
I want to leave the more in-depth explanation for the README and have the inline comments be short & sweet.

Do you still want me to modify the comment or can I just include this in the README?

Reply:
That's fine.